bodhi/server/captcha.py

```diff
@@ -29,7 +29,7 @@
 
 def math_generator(plainkey, settings):
     """ Given a plainkey, return its expected value.
-    If plainkey is None, return a random one.
+    If plainkey is None, return a random one. settings is not used.
```

Comment:
I've learned the use of the ``_`` argument which allows to do something like:
````
>>> def foo(bar, _):
...     print bar
>>> foo(1)
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    foo(1)
TypeError: foo() takes exactly 2 arguments (1 given)
>>> foo(1, 2)
1
````

Basically, you require two arguments but you never use the second one (marked as `_`) and code lint such as pep8 or pylint won't complain about them.